src/scanner.cc

```diff
@@ -75,14 +77,87 @@ struct Scanner {
     }
   }
 
-  // Scans the multistring start. Assumes that the m indicating a multistring
-  // was already consumed.
+  bool valid_symtag_start(int32_t lookahead) {
```

Comment:
Might deserve a one-liner comment, I think "symtag" could not be obvious in a distant future.